### DISCUSSION TITLE
Silent mode

### DISCUSSION THREAD

**Discussion by lorbetzki** [revk/xESP32-Faikout#97] [Ideas]
Hello,

the silent mode works very well, but i have a little problem with it... [here](https://github.com/revk/ESP32-Faikin/issues/34) is described, that the Daikin dont report the mode back.

With the original system, the night mode is activated and remains at a stop and restart.

Is it possible to build a button "silent mode" that, when active, always puts the system in silent mode? Maybe we can check the rpm of the fan? i my case the system is between 510 and 520 rpm in silent mode. 

this would be a very "cool" feature, because we use this mode in our bedrooms.

Best regards.

**Comment by revk** [comment]
Ok I see why. I'll try and work out the logic maybe.

I suspect we could have a setting that whenever we see it report a change to quiet mode we set it to night mode. So there is only night mode. Or whatever it is called. And we don't then even have a separate button.

**Comment by revk** [comment]
OK, not sure if this is quite what you want, but upgrade to latest code, and it should have a new setting (in settings page) for fan mode always night/quiet, not auto. Let me know if that does what you want.

**Comment by revk** [comment]
Or maybe that is not what you mean? I guess maybe fan could be a clue. What fan speed is the lowest for real "auto"?

**Comment by revk** [comment]
OK scrapped that setting, working on using fan rpm as guide.

**Comment by lorbetzki** [comment]
not what I expected, but the direction is right.

Automatic should still be selectable, I was thinking more of something that if the last mode you have selected on the Faikin is also the one that will be selected again at the next start, **even** for silent. 

I know the problem that we do not get an answer, but the faikin knows if and when I activate it.

If i choose Auto, the slowest speed in my case is between 650/660rpm and corresponds to Fan speed 1

**Comment by revk** [comment]
Yes, I see what you are getting at now, will have code release in a moment.

**Comment by revk** [comment]
Ok new code, but it does mean it is a tad confused as it sees the low fan when at target temperature as being night not auto... So not 100% sure on this. If it confuses people it may end up being a separate setting.
